An error occurred while executing the following cell:
------------------
# SVI for a mixture of 3 Gaussians in 2d
# https://github.com/brendanhasz/svi-gaussian-mixture-model/blob/master/BayesianGaussianMixtureModel.ipynb

import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

try:
    import tensorflow as tf
except ModuleNotFoundError:
    %pip install -qq tensorflow
    import tensorflow as tf
try:
    import tensorflow_probability as tfp
except ModuleNotFoundError:
    %pip install -qq tensorflow-probability
    import tensorflow_probability as tfp
tfd = tfp.distributions

try:
    import probml_utils.svi_gmm_model_tfp as GMM
except ModuleNotFoundError:
    %pip install -qq svi_gmm_model_tfp
    import probml_utils.svi_gmm_model_tfp as GMM

from probml_utils import savefig
import numpy as np
import matplotlib.pyplot as plt
import os
------------------

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/tmp/ipykernel_3777/3123412331.py in <module>
     20 try:
---> 21     import probml_utils.svi_gmm_model_tfp as GMM
     22 except ModuleNotFoundError:

ModuleNotFoundError: No module named 'probml_utils'

During handling of the above exception, another exception occurred:

ModuleNotFoundError                       Traceback (most recent call last)
/tmp/ipykernel_3777/3123412331.py in <module>
     22 except ModuleNotFoundError:
     23     get_ipython().run_line_magic('pip', 'install -qq svi_gmm_model_tfp')
---> 24     import probml_utils.svi_gmm_model_tfp as GMM
     25 
     26 from probml_utils import savefig

ModuleNotFoundError: No module named 'probml_utils'
ModuleNotFoundError: No module named 'probml_utils'
